### PR TITLE
enhance: add milvus client rbac test cases

### PR DESCRIPTION
**Pull request by NicoYuan1986**
link: #30221

### CODE REVIEW

File: tests/python_client/base/high_level_api_wrapper.py [hunk 1]
```diff
@@ -523,4 +537,144 @@ def using_database(self, client, db_name, timeout=None, check_task=None, check_i
                                        check_items, check,
                                        db_name=db_name,
                                        **kwargs).run()
-        return res, check_result
\ No newline at end of file
+        return res, check_result
+
```
Comment (binbinlv) [inline tests/python_client/base/high_level_api_wrapper.py:541]: add trace () here please.

Comment (NicoYuan1986) [reply tests/python_client/base/high_level_api_wrapper.py:541]: OK. I will modify in the next pr.

**Pull request merged**